README.md

```diff
@@ -141,10 +141,11 @@ In order to use **semantic-release** you need:
 
 Let people know that your package is published using **semantic-release** by including this badge in your readme.
 
-[![semantic-release](https://img.shields.io/badge/semantic-release-e10079.svg?logo=semantic-release)](https://github.com/semantic-release/semantic-release)
+[![semantic-release: conventionalcommits](https://img.shields.io/badge/semantic--release-conventionalcommits-e10079?logo=semantic-release)](https://github.com/semantic-release/semantic-release)
```

Comment:
since the default preset is `angular` i think we should suggest that one here too. we do plan to update the default to `conventional-commits` at some point, but we can update that here at that point

Reply:
Fixed too.